src/test/java/com/github/dockerjava/api/command/CommandJSONSamples.java

```diff
@@ -24,7 +24,7 @@
  */
 public enum CommandJSONSamples implements JSONResourceRef {
 
-    inspectContainerResponse_full, inspectContainerResponse_full_1_21, inspectContainerResponse_empty;
+    inspectContainerResponse_full, inspectContainerResponse_full_1_21, inspectContainerResponse_empty, inspectContainerResponse_full_healthcheck;
```

Comment:
old deprecated way of testing. Please use test files directly via helper methods as it done in https://github.com/docker-java/docker-java/blob/419900d074e6d9754dcd5c6290d93c3350bab010/src/test/java/com/github/dockerjava/api/model/EventsTest.java#L25